### DISCUSSION TITLE
Getting "Label for shipment cannot be found" when requesting Pallet Labels for an LTL Shipment

### DISCUSSION THREAD

**Discussion by rrff123** [amzn/selling-partner-api-models#3631] [General]
When using the Fulfillment Inbound API /getLabels call requesting LabelType=PALLET it returns the following error message:

{  "errors": [    {      "code": "InvalidInput",      "message": "Label for shipment cannot be found!",      "details": ""    }  ]}

If in Seller Central the Shipment is converted pallet labels can be printed manually. Anyone having the same issue?

**Comment by abhishek-hopstack** [comment]
i am having the same issue. do you have any solution for this @rrff123

**Comment by haoxi911** [comment]
Same here. Not sure what is the root cause.

**Reply by haoxi911** [reply]
Have you figured it out yet? @rrff123 @abhishek-hopstack

**Reply by dmelendez35** [reply]
Did somebody figured it out yet? I'm having the same issue

**Reply by VarunKumarKishore** [reply]
I am facing the same issue. Please let me know if you have found a solution.
